### PR TITLE
Update CITATION.cff

### PR DESCRIPTION
**Pull request by amjjbonvin**
Updating the citation information

### CODE REVIEW

**Review by VGPReys — APPROVED** [review]

**Comment by VGPReys** [comment]
Will this solve the zenodo issue ?

**Comment by amjjbonvin** [comment]
I suggest we create a new release, following the naming of the previous one, i.e. 3.0.0.beta.6

File: CITATION.cff [hunk 1]
```diff
@@ -1,54 +1,43 @@
...
+  given-names: João
+  affiliation: Zymvol
+- family-names: van Noort
+  given-names: Charlotte
+  affiliation: Utrecht University
+- family-names: Verhoeven
+  given-names: Stefan
+  affiliation: Netherlands eScience Center
+- family-names: Versini
+  given-names: Raphaella
```
Comment (mgiulini) [inline CITATION.cff:36]:
```suggestion
  given-names: Raphaelle
```

**Review by mgiulini — APPROVED** [review]

**Comment by mgiulini** [comment]
> I suggest we create a new release, following the naming of the previous one, i.e. 3.0.0.beta.6

i can do it as soon as this gets merged

**Review by VGPReys — APPROVED** [review]

**Pull request closed without merging**